cmd/vela-worker/server.go

```diff
@@ -56,10 +57,35 @@ func (w *Worker) server() (http.Handler, bool) {
 			logrus.Fatal("unable to run with TLS: No certificate provided")
 		}
 
-		return _server, true
+		// define TLS config struct for server start up
+		tlsCfg := &tls.Config{
+			MinVersion: tls.VersionTLS12,
```

Comment:
now with the default set where the flags are defined, this feels unnecessary. thoughts?